src/software/new_geom/util/distance_test.cpp

```diff
@@ -104,15 +104,6 @@ TEST(DistanceTest, point_off_segment_closest_to_segment_end)
     EXPECT_DOUBLE_EQ(distance(s, p), expected);
 }
 
-TEST(DistanceTest, point_off_degenerate_segment)
-{
-    Point p(5, 3);
-    Segment s(Point(-2, 2), Point(-2, 2));
-    double expected = std::hypot(7, 1);
-    EXPECT_DOUBLE_EQ(distance(p, s), expected);
-    EXPECT_DOUBLE_EQ(distance(s, p), expected);
-}
```

Comment:
Sorry, why did we remove this test?

Reply:
This was removed because creating a segment from (-2, 2) to (-2, 2) would result in a degenerate segment and throw an error.

Reply:
Added test back in with a nearly degen segment.

Reply:
Just to double check, we don't need to remove this now because we permit 0-length segments?

Reply:
This is correct.